<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QDialog" name="Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Find
{
	border-image: url(:/new/prefix1/background.jpg);
}
QDialog#Find QPushButton
{
	border-image: url(:/new/prefix1/background.jpg);
	border-radius:8px;
}
QDialog#Find QPushButton:hover
{
	
	border-image: url(:/new/prefix1/backgroundp.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label_find">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>135</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_new">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>265</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_mail">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>395</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>QQ邮箱</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_icode">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>验证码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>670</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_change">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>670</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_user">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_new">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入新密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mail">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>请输入邮箱</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_icode">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>260</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>请输入验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_geticode">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>530</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
